include/extractor/profile_properties.hpp

```diff
@@ -12,11 +14,18 @@ const constexpr auto DEFAULT_MAX_SPEED = 180 / 3.6; // 180kmph -> m/s
 
 struct ProfileProperties
 {
+    enum
```

Comment:
Hrm just looked this up because I was sure I had seen stuff like `static const constexpr std::size_t value = 123;` in class definitions before.

I'm trying to interpret [this](http://stackoverflow.com/a/34053854/1944243) comment. From my reading this would allow for doing this as long as the `static const constexpr` member is of integral type.

Is my reading of the standard correct here?